File: example/javalib/testing/3-test-deps/baz/test/src/baz/BazTestUtils.java
package baz;

public class BazTestUtils {

  public static void bazAssertEquals(Object x, Object y) {
    System.out.println("Using BazTestUtils.bazAssertEquals");
    if (!x.equals(y)) {
      throw new AssertionError("Expected " + y + " but got " + x);
    }
  }
}

File: example/kotlinlib/testing/3-test-deps/baz/test/src/baz/BazTestUtils.kt
package baz

object BazTestUtils {
    fun bazAssertEquals(
        x: Any,
        y: Any,
    ) {
        println("Using BazTestUtils.bazAssertEquals")
        if (x != y) {
            throw AssertionError("Expected $y, but got $x")
        }
    }
}
